cpp/lib/include/IssnLookup.h

```diff
@@ -0,0 +1,61 @@
+/** \file   IssnLookup.h
+ *  \brief   The Utility for extracting issn information from https://portal.issn.org/
+ *  \author  Steven Lolong (steven.lolong@uni-tuebingen.de)
+ *
+ *  \copyright 2023 Tübingen University Library.  All rights reserved.
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <iostream>
+#include <nlohmann/json.hpp>
```

Comment:
Since the user of this interface will not interact with the json library at all, you might as well remove the include here so it's only included in IssnLookup.cc